components/clm/src/biogeophys/WaterStateType.F90

```diff
@@ -670,7 +670,7 @@ subroutine InitCold(this, bounds, &
                   if (j > nlevbed) then
                      this%h2osoi_vol_col(c,j) = 0.0_r8
                   else
-                     this%h2osoi_vol_col(c,j) = 0.15_r8
+                     this%h2osoi_vol_col(c,j) = 0.70*watsat_col(c,j)  !0.15_r8 to avoid very dry conditions that cause errors in FATES HYDRO
```

Comment:
I suggest initializating soil moisture to a different value only when FATES HYDRO is active via:

```
if (use_fates_planthydro) then
   this%h2osoi_vol_col(c,j) = 0.70_r8*watsat_col(c,j) 
else
   this%h2osoi_vol_col(c,j) = 0.15_r8
endif
```